plugins/servicenow/plugin.spec.yaml

```diff
@@ -627,6 +644,202 @@ actions:
+        type: string
+        required: false
+        example: Example technical details about the vulnerability
+      additional_fields:
+        title: Additional Fields
+        description: JSON object containing the additional fields and values to create incident
+        type: object
+        required: false
+        example: '{"description": "Example description"}'
+    output:
+      system_id:
+        title: System ID
+        description: System ID of the new vulnerability created
+        type: string
+        required: true
+        example: 9de5069c5afe602b2ea0a04b66beb2c0
+      number:
+        title: Vulnerability Number
+        description: Vulnerability ticket number
+        type: string
+        required: true
+        example: 1
+      vulnerability_url:
+        title: Vulnerability URL
+        description: URL to newly created vulnerability
+        type: string
+        required: true
+        example: "https://example.service-now.com/sn_vul_app_vulnerable_item.do?sys_id=61..."
+  get_vulnerability:
+    title: Get Vulnerability
+    description: Retrieve the vulnerability by ID
+    input:
+      system_id:
+        title: System ID
+        description: System ID of the vulnerability to be retrieved
+        type: string
+        required: true
+        example: 9de5069c5afe602b2ea0a04b66beb2c0
+      filtering_fields:
+        title: Filtering Fields
+        description: Comma-separated list of fields desired in output object (e.g.
+          opened_by,number)
+        type: string
+        required: true
+        example: opened_by,number
+    output:
+      filtered_vulnerability:
+        title: Filtered Vulnerability
+        description: JSON object representing the vulnerability containing the given fields
+        type: object
+        required: true
+        example: '{"number":"1","opened_by":{"link":"https://example.service-now.com/api/now/table/sys...","value":""},"state":"1"}'
+  update_vulnerability:
+    title: Update Vulnerability
+    description: Update the vulnerability by ID
+    input:
+      system_id:
+        title: System ID
+        description: System ID of the vulnerability to be retrieved
+        type: string
+        required: true
+        example: 9de5069c5afe602b2ea0a04b66beb2c0
+      risk_rating:
+        title: Risk Rating
+        description: The risk rating of the vulnerability
+        type: string
+        enum:
+          - Low
+          - Medium
+          - High
+          - Critical
+          - ""
+        required: false
+        example: SAFE
+      state:
+        title: State
+        description: The state of the vulnerability
+        type: string
+        enum:
+          - Open
+          - Under Investigation
+        required: false
+        example: Open
+      first_found:
+        title: First Found
+        description: The time, in ISO format, that represents the vulnerability was first found
```

Comment:
(Reword):  The time that represents the vulnerability was first found,  in ISO format